src/Illuminate/Auth/Access/Gate.php

```diff
@@ -162,7 +164,7 @@ public function check($ability, $arguments = [])
      *
      * @param  string  $ability
      * @param  array  $arguments
-     * @return callable
+     * @return mixed
```

Comment:
It's not.
In one case call_user_func_array takes a callable as first argument, in the other case it takes an array